### PR TITLE
fix: check if IP is in subnet

### PR DESCRIPTION
**Pull request by omBratteng**
I've checked that it finds the IP in the array of IPs

### CODE REVIEW

**Review by vpol — APPROVED** [review]

**Review by idoshamun — APPROVED** [review]

File: .infra/Pulumi.adhoc.yaml [hunk 1]
```diff
@@ -82,6 +82,5 @@ config:
     typeormUser: postgres
     tz: UTC
     urlPrefix: http://api.local.com
-    vordrIps: ""
```
Comment (capJavert) [inline .infra/Pulumi.adhoc.yaml:85]: why removed?

Comment (omBratteng) [reply .infra/Pulumi.adhoc.yaml:85]: `process.env.VORDR_IPS?.split(',')` returns `['']`, which makes the `isInSubnet` subnet fail. So if I remove it, it falls back to `[]`

Comment (capJavert) [reply .infra/Pulumi.adhoc.yaml:85]: add `filter(Boolean)`?

Comment (omBratteng) [reply .infra/Pulumi.adhoc.yaml]: Yeah, just did that now

**Review by idoshamun — APPROVED** [review]

**Pull request merged**